lti_consumer/templatetags/lti_sanitize.py

```diff
@@ -13,7 +13,7 @@ def lti_sanitize(html):
     """
     Sanitize a html fragment with bleach.
     """
-    allowed_tags = bleach.sanitizer.ALLOWED_TAGS + ['img']
+    allowed_tags = bleach.sanitizer.ALLOWED_TAGS | {'img'}
```

Comment:
Won't this break in `edx-platform`? According to [the changelog](https://bleach.readthedocs.io/en/latest/changes.html#version-6-0-0-january-23rd-2023), the list -> dict change was introduced in `bleach==6.0.0`.
`edx-platform` is still using [bleach[CSS]==5.0.1](https://github.com/openedx/edx-platform/blob/e576478/requirements/edx/base.txt#L61) because it is pinned [here](https://github.com/openedx/edx-platform/blob/e576478/requirements/constraints.txt#L79). If we use the newer version in the CI, then we won't catch any incompatibilities.

Reply:
Done, thanks for the review!

Reply:
@feanil, please see [my suggestion from the Drag and Drop v2 XBlock](https://github.com/openedx/xblock-drag-and-drop-v2/pull/308#discussion_r1092974520) before merging this.

Reply:
@Agrendalath sorry I missed this comment, let me know if you think it's worth it to make this backwards compatible, I'm happy to put up or review a PR to that effect.

Reply:
@feanil, no worries. Since the major version is already released, let's keep it as is.